packages/fury-adapter-swagger/test/schema-test.js

```diff
@@ -5,6 +5,7 @@ const { DataStructureGenerator } = require('../lib/schema');
 const namespace = new Fury().minim;
 
 const {
+  Element,
   String: StringElement,
   Number: NumberElement,
   Boolean: BooleanElement,
```

Comment:
Side note, I wouldn't rename the classes like this, not even just for the purpose of the tests. It's not that unusual one needs to use JavaScript's builtin `String`, `Number`, etc. types, and this makes it impossible. Reading the tests can also be confusing (_is this a JavaScript's String or is this a minim element?_)

Reply:
We're renaming the elements from what is stored in the namespaces `elements` collection. The namespace elements collection is named as they are in Refract Spec "Number", "String", "Boolean", "HttpRequest", "Resource" etc. As these names clash in JavaScript land we're calling them `NumberElement`, `StringElement` etc. JavaScript doesn't have a namespace system, in C++ or such this is clearer because it can be such as `refract::String` where `refract` is a namespace for the structures.

Reply:
I find the JS syntax for this a little hard to remember as I think of it as backwards. It's effectively:

```js
const StringElement = minim.elements['String'];
```